envs/env-py39.yml

```diff
@@ -75,7 +75,7 @@ dependencies:
   - nslsii >=0.8.0
```

Comment:
Could we also bump the `nslsii` version to be [0.9.0](https://github.com/NSLS-II/nslsii/releases/tag/v0.9.0)? (this applies to the py310 file too.)

Reply:
done in https://github.com/nsls2-conda-envs/nsls2-collection-tiled/pull/11/commits/202e38e7b0e8034e2585d3b582776f43aca9872b